src/execution_plan/ops/op_create.c

```diff
@@ -237,20 +238,21 @@ static void _CommitEdges(OpCreate *op) {
 	op->stats->relationships_created += relationships_created;
 }
 
-static void _CommitNewEntities(OpCreate *op) {
+static bool _CommitNewEntities(OpCreate *op) {
```

Comment:
Please explain this change, should we issue a run-time exception if we're unable to lock?

Reply:
we are issuing a runtime error, at `query_ctx.c`
this is the fallback incase the runtime error falis.